### PR TITLE
Update Dockerfile to install Yarn, fastlane using bundler

### PR DESCRIPTION
**Pull request by mayankkapoor**
* Install yarn package manager
* Use bundle install to install fastlane and dependencies, Add Gemfile (#1)
* Add Gemfile, update Docker to install fastlane & dependencies via bundle install
* Make /.fastlane folder
* Update Dockerfile to add ENV BUNDLE_GEMFILE

After testing Android build using fastlane, the default fastlane installed on the image used to fail. The `/.fastlane` folder wasn't writable, and fastlane kept missing dependencies. So I installed fastlane using bundler to take care of the dependencies. I also added the yarn package manager which quite a few teams use instead of npm. Feedback welcome.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -181,15 +184,24 @@ RUN echo "Installing sdk tools ${ANDROID_SDK_TOOLS_VERSION}" && \
...
-    mkdir -p /home/jenkins && \
-    chmod 777 /home/jenkins && \
-    chmod 777 /var/lib/jenkins/workspace && \
-    chmod 777 $ANDROID_HOME/.android
-
+RUN mkdir -p /var/lib/jenkins/workspace
+RUN mkdir -p /home/jenkins
+RUN chmod 777 /home/jenkins
+RUN chmod 777 /var/lib/jenkins/workspace
+RUN chmod 777 $ANDROID_HOME/.android
```
Comment (mingchen) [inline Dockerfile]: Please use single `RUN` to reduce docker layers

Comment (mayankkapoor) [reply Dockerfile]: @mingchen Done.

**Comment by mingchen** [comment]
Thanks for your contribution. @mayankkapoor

**Comment by mingchen** [comment]
Change included in tag: `1.15.0`

**Pull request merged**